docs/enterprise-guide/authenticating-with-saml.md

```diff
@@ -79,6 +79,14 @@ name), last name (surname), and email address are included as
 attributes of the first assertion returned in the identity provider's
 SAML response.
 
+We've pulled the attributes out of the XML above for easy copy/pasting into your SAML identity provider:
+
+| Name      | Value |
```

Comment:
Nit-picky, but I might change the `Name` column heading to `Attribute`.

Reply:
OKTA labels the field as Name for some reason, so I think it makes sense to keep it consistent, though I agree that attribute would likely make more sense overall!

Reply:
New screenshot looks correct for me, but could we also explicitly call it out rather than just use the screen shot in case someone is using a different identity provider?

Reply:
Rereading this section, it kind of fails to explain how to actually configure anything, the purpose of this screenshot isn't even to show that initial field, it's just a bonus. I'm going to rewrite more of it.